pkg/cli/cvd.go

```diff
@@ -87,29 +91,54 @@ func newCVDCommand(opts *subCommandOpts) *cobra.Command {
 func createCVD(c *cobra.Command, flags *CreateCVDFlags, opts *subCommandOpts) error {
 	service, err := opts.ServiceBuilder(flags.CommonSubcmdFlags, c)
 	if err != nil {
-		return err
+		return fmt.Errorf("Failed to build service instance: %w", err)
+
 	}
-	if flags.LocalImage {
-		return createCVDFromLocalBuild(service, c, flags)
+	action := &createCVDAction{
+		Service: service,
+		Opts:    *flags.CreateCVDOpts,
+	}
+	cvd, err := action.Execute()
+	if err != nil {
+		return fmt.Errorf("Failed to create cvd: %w", err)
+	}
+	output := CVDOutput{
+		BaseURL: buildBaseURL(flags.CVDRemoteFlags),
+		Host:    flags.Host,
+		CVD:     cvd,
+	}
+	c.Printf("%s\n", output.String())
+	return nil
+}
+
+type createCVDAction struct {
```

Comment:
nit: Instead of `createCVDAction` and `Execute`, you could have `CVDCreator` and `Create` respectively.

Reply:
`CVDCreator` seems appropriate, however the application layer will have many action so I was looking for a naming convention for the classes in the frontier of the application layer, i.e `listCVDsAction`, `deleteCVDAction`, `deleteHostAction` and so on.

Reply:
Unless you are going to put a bunch of different actions together in a collection and therefore make them all implement an `Action` interface; there is no need to have them all share that name. Introducing polymorphism here doesn't seem to help, but it still has the disadvantage of reducing readability (example: `Execute` is not as readable as `CreateCVD`).

Reply:
acknowledged, done